prime-router/src/main/kotlin/cli/tests/Hl7Tests.kt

```diff
@@ -18,6 +18,7 @@ class Hl7Ingest : CoolTest() {
     override val status = TestStatus.SMOKE
 
     override fun run(environment: ReportStreamEnv, options: CoolTestOptions): Boolean {
+        initListOfGoodReceiversAndCounties(environment)
```

Comment:
List of receivers to test is now dependent on the Environment.  If not in staging, include REDOX.